apolloschurchapp/src/content-single/ContentSingle.js

```diff
@@ -9,16 +9,22 @@ import {
   ThemeMixinConnected,
 } from '@apollosproject/ui-connected';
 
+import { styled } from '@apollosproject/ui-kit';
+
 import ActionContainer from './ActionContainer';
 
+const NodeSingleConnectedWithActionContainer = styled(
+  ({ theme: { sizing } }) => ({ paddingBottom: sizing.baseUnit * 5 })
+)(NodeSingleConnected);
+
 const ContentSingle = (props) => {
   const nodeId = props.route?.params?.itemId;
   const { data, loading } = useQuery(
     gql`
-      query getContentNodeTitle($nodeId: ID!) {
+      query getContentNodeTitle($nodeId: ID) {
```

Comment:
what's this change for?

Reply:
Ah, I think that's a merge conflict. The change below (id being moved) fixes that query and stops it form erroring (ContentNode doesn't have an ID property)